anotherfile.js

```diff
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
+    case "add":
+      return a + b;
+    case "subtract":
+      return a - b;
+    case "multiply":
+      return a * b;
+    case "divide":
+      if (b === 0) return "Error: Cannot divide by zero";
```

Comment:
Returning a string for an error case in a function that otherwise returns numbers can lead to inconsistent return types, which may cause issues for consumers of this function.

**Suggestion:**
```suggestion
Throw an error or return a consistent error object instead of a string to maintain consistent return types.
```

**Context:**
```
if (b === 0) return "Error: Cannot divide by zero";
```